jsonschema/validators.py

```diff
@@ -522,15 +523,15 @@ class RefResolver(object):
             A mapping from URI schemes to functions that should be used
             to retrieve them
 
-        urljoin_cache (functools.lru_cache):
+        urijoin_cache (functools.lru_cache):
```

Comment:
This would need to stay as-is for backwards compatibility.

Not sure the URL/URI distinction is too important anymore anyhow.

Reply:
Sure, I made a judgement that it wasn't used internally so it could be removed, in hindsight not a good decision.